plugins/GoogleSearchConsole/v1/docs/README.md

```
@@ -100,4 +109,5 @@ Verify that:
 
 - The property exists in Google Search Console
 - The signed-in Google account has access to the property
-- The property URL exactly matches the URL entered in the plugin configuration
\ No newline at end of file
+- The property value exactly matches the identifier in Search Console
+- A **403 "insufficient permission"** despite being an owner usually means the property *type* is wrong — e.g. entering `https://example.com/` for a **Domain** property. Use `sc-domain:example.com` instead (and vice versa)
```

markdownlint-cli2 (0.23.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)